task_03.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""This is a mortgage calculator. Lifetime compound interest of loan."""
+
+import decimal
+
+
+NAME = raw_input('What is your name? ')
+PRINC = int(raw_input('What is the amount of your principal? '))
+DURAT = int(raw_input('For how many years is this loan being borrowed? '))
+PREQU = raw_input('Are you pre-qualified for this loan? ')
+PREQU1 = PREQU[:1]
+PREQU2 = PREQU1.lower()[:1]
+
+if PRINC <= 199999:
+    if DURAT <= 15:
```

Comment:
The safe course here would have been to guard against values that could have been less than 0 -- which is definitely what happened. Bad user input is something against which we must always guard. Here someone had negative values for principal and/or duration.